tutorials/tpot_mnist_pipeline.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import pandas as pd
+
+from sklearn.cross_validation import StratifiedShuffleSplit
+from itertools import combinations
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.neighbors import KNeighborsClassifier
+
+# NOTE: Make sure that the class is labeled 'class' in the data file
+tpot_data = pd.read_csv('PATH/TO/DATA/FILE', sep='COLUMN_SEPARATOR')
+training_indeces, testing_indeces = next(iter(StratifiedShuffleSplit(tpot_data['class'].values, n_iter=1, train_size=0.75)))
```

Comment:
Please ensure that `test_size=0.25` is also in this call. See #52 for the reasoning.